### ISSUE TITLE
Filter box is partially hidden, v2.0.15

### ISSUE THREAD

**Issue by supamanda** [adazzle/react-data-grid#673]
> **HAVE YOU ALREADY SEARCHED FOR SIMILAR ISSUES? PLEASE HELP US OUT AND DOUBLE-CHECK FIRST!**

> **ALSO, PLEASE DON'T BE THAT PERSON WHO DELETES THIS TEMPLATE. IT'S HERE FOR A REASON.**

> **THANKS!**

**WHICH VERSION OF REACT ARE YOU USING?**
```
Officially Supported:
[ ] v0.14.x
```

```
Community Supported:
[x ] v15.0.x
```

**WHICH BROWSER ARE YOU USING?**

```
Officially Supported:
[ ] IE 9 / IE 10 / IE 11
[ ] Edge
[x ] Chrome
```

```
Should work:
[ ] Firefox
[ ] Safari
```

--------

**I'm submitting a ...**  (check one with "x")
```
[x ] bug report
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/adazzle/react-data-grid/blob/master/CONTRIBUTING.md
```

**Current behavior**
Filter text box row is partially obscured by the data

**Expected/desired behavior**
Filter text box row is fully visible

**Reproduction of the problem**
If the current behavior is a bug or you can illustrate your feature request better with an example, please provide the steps to reproduce and if possible a minimal demo of the problem.

1. Go to http://adazzle.github.io/react-data-grid/examples.html#/filterable-grid
2. Click on 'Filter rows' button
![partially_hidden_filter](https://cloud.githubusercontent.com/assets/3926025/23193879/26447c2e-f8ff-11e6-8dd0-b56c92ae11fe.jpg)

**What is the expected behavior?**

**What is the motivation / use case for changing the behavior?**

**Comment by supamanda** [comment]
The workaround I'm using is to explicitly set the headerRowHeight and the headerFiltersHeight, ie
```
<ReactDataGrid
  ...
  headerRowHeight={35}
  headerFiltersHeight={45}
  ... />
```

**Comment by supamanda** [comment]
Looks like this was fixed in 2.0.17, with #683   Thanks @joao-inacio  :)
